supabase/migrations/20230821125643_telegram_tables_rls.sql

```diff
@@ -1 +1,3 @@
-ALTER TABLE telegram_bot_groups ENABLE ROW LEVEL SECURITY;
\ No newline at end of file
+ALTER TABLE telegram_bot_groups ENABLE ROW LEVEL SECURITY;
+ALTER TABLE sessions ENABLE ROW LEVEL SECURITY;
+ALTER TABLE telegram_users ENABLE ROW LEVEL SECURITY;
```

Comment:
The supabase migration is failing, probably because it was modified manually. CLI should be used.   
![image](https://github.com/ubiquity/telegram-ubiquibot/assets/78043783/c4807d6e-2470-40f1-a6b5-6988a82dd01a)

Reply:
I'd add that rn, so you can test again

Reply:
fixed here https://github.com/ubiquity/telegram-ubiquibot/pull/17/commits/721280ce49bdc5dd98bd490ac64fddce293e544f